scripts/machine-setup.sh

```diff
@@ -5,7 +5,7 @@ sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ xenial main'
 sudo apt-add-repository 'deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main'
 
 sudo apt-get update
-sudo apt-get install clang-8 cmake yasm -y
+sudo apt-get install clang-8 cmake g++-8 yasm -y
```

Comment:
Some C++17 features, like `filesystem`, weren't part of `libstdc++` until `G++ 8`